daemon/cluster/executor/container/controller.go

```diff
@@ -185,7 +185,7 @@ func (r *controller) Prepare(ctx context.Context) error {
 }
 
 // Start the container. An error will be returned if the container is already started.
-func (r *controller) Start(ctx context.Context) error {
+func (r *controller) Start(ctx context.Context) (err error) {
```

Comment:
Could you rename `err` to `retErr` to make it stand out as being this output-variable? `err` is used as name for local variables in this function, which makes it easy to confuse them (and to accidentally mask the output variable)

Reply:
yes, I will rename it to make it clearer.